src/azul/terraform.py

```diff
@@ -708,6 +708,10 @@ def tf_config(self, app_name):
         for resource in resources['aws_lambda_function'].values():
             assert 'layers' not in resource
             resource['layers'] = ['${aws_lambda_layer_version.dependencies.arn}']
+            # Publishing the Lambda function as a new version prevents possible
+            # race conditions when there's a cyclic dependency between an update
```

Comment:
```suggestion
            # race conditions when there's a dependency between an update
```

Reply:
It doesn't have to be cyclic. And for graphs of two vertices, "mutual" would be a better word.